recipes/python-soxr/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "soxr" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: python-soxr
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/soxr-{{ version }}.tar.gz
+  sha256: 8772c1482683768843074097d5363461d781cd6168619cc7815cb728e5c901ee
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+    - setuptools_scm
```

```suggestion
    - setuptools_scm
    - soxr
```
Add soxr to the host requirements section, then patch your build to dynamically link against the pre-compiled libsoxr instead of building it again.